frontend/app-development/layout/PageHeader.tsx

```diff
@@ -11,14 +11,25 @@ import { TopBarMenu } from './AppBar/appBarConfig';
 import { useTranslation } from 'react-i18next';
 import { AltinnButtonActionItem } from 'app-shared/components/altinnHeader/types';
 import { GiteaHeader } from 'app-shared/components/GiteaHeader';
+import { SettingsModalButton } from './SettingsModalButton';
+import { shouldDisplayFeature } from 'app-shared/utils/featureToggleUtils';
 
 type SubMenuContentProps = {
   org: string;
   app: string;
 };
 
 export const subMenuContent = ({ org, app }: SubMenuContentProps) => {
-  return <GiteaHeader org={org} app={app} hasCloneModal />;
+  return (
+    <GiteaHeader
+      org={org}
+      app={app}
+      hasCloneModal
+      leftComponent={
+        shouldDisplayFeature('settingsModal') && <SettingsModalButton org={org} app={app} />
```

Comment:
@framitdavid, is this the correct way to add the feature flag? 😄

Reply:
Looks OK, if the feature-flag is not set the `leftComponent `will become `undefined`. 😄